qa/os/build.gradle

```diff
@@ -62,7 +62,9 @@ tasks.register('destructivePackagingTest') {
 }
 
 processTestResources {
-  from project(":test:framework").file("src/main/resources/log4j2-test.properties")
+  from(project(":test:framework").file("src/main/resources/log4j2-test.properties")) {
```

Comment:
Should we even need to share this config? Why not just have a separate config for these tests? Also, so far as I can tell, we don't really use anything from `:test:framework` anyhow. All packaging tests seem to rely only on vanilla junit and randomized runner.

Reply:
Aye, I was just being really cautious, but it's probably unnecessary. I've replaced it with a dedicated file.